lib/cassandra_migrations/railtie/tasks.rake

```diff
@@ -65,10 +67,10 @@ namespace :cassandra do
   end
 
   namespace :test do
-    desc 'Load the development schema in to the test keyspace'
+    desc 'Load the development schema in to the test keyspace via a full reset'
     task :prepare do
       Rails.env = 'test'
-      Rake::Task['cassandra:reset'].execute
+      Rake::Task['cassandra:migrate:reset'].execute
```

Comment:
my thoughts here are that `cassandra:reset` (a la rails `db:reset`) implies that loading from schema (not migrations) and seeding is occuring. Wheras `migrate:reset` implies what we're actually doing (drop, create, migrate).

http://jacopretorius.net/2014/02/all-rails-db-rake-tasks-and-what-they-do.html